src/objects.js

```diff
@@ -3041,6 +3038,11 @@ SpriteMorph.prototype.freshPalette = function (category) {
                     ]
             };
 
+        if (category === 'unified') {
+            more.unified = Object.values(more).reduce((x, y) =>
+                x.concat(y));
+        }
+
         function hasHiddenPrimitives() {
             var defs = SpriteMorph.prototype.blocks,
                 hiddens = StageMorph.prototype.hiddenPrimitives;
```

Comment:
Probably not worth re-writing, but I wonder if it would be more clear to iterate over blocksForCategory rather than all blocks...